### PR TITLE
 DMP-3514 Re-enable tests relying on Redis testcontainer

### PR DESCRIPTION
**Pull request by mario-paniccia**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-3514

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBase.java [hunk 1]
```diff
@@ -10,28 +10,23 @@
...
+import org.testcontainers.containers.GenericContainer;
 import uk.gov.hmcts.darts.test.common.LogUtil;
 import uk.gov.hmcts.darts.test.common.MemoryLogAppender;
 import uk.gov.hmcts.darts.testutils.stubs.DartsDatabaseStub;
 
 import java.util.List;
 
 /**
- * Base class for integration tests running with H2 in PostgreSQL compatibility mode.
+ * Base class for integration tests running with H2 in Postgres compatibility mode.
```
Comment (davet1985) [inline src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBase.java:23]: Not sure this comment is needed here?

Comment (mario-paniccia) [reply src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBase.java:23]: I have this comment to distinguish this class with `PostgresIntegrationBase` which has the following comment:
```
/**
 * Base class for integration tests running against a containerized Postgres with Testcontainers.
 */
```
so it's clear when to use `PostgresIntegrationBase` vs `IntegrationBase`. 
But I can remove it if it's unnecessary

Comment (davet1985) [reply src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBase.java:23]: Ah ok, I get it, that's cool. Is it worth mentioning that using this class runs redis as well?

Comment (mario-paniccia) [reply src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBase.java:23]: I have noted this down and will change the comment as part of a next PR of mine. So I don't have to wait for a new build

**Review by davet1985 — APPROVED** [review]

**Pull request merged**